core/clipboard.js

```diff
@@ -0,0 +1,44 @@
+/*
+ * noVNC: HTML5 VNC client
+ * Copyright (c) 2025 Tobias Fahleson
+ * Licensed under MPL 2.0 or any later version (see LICENSE.txt)
+ */
+
+import * as Log from './util/logging.js';
+
+export default class Clipboard {
```

Comment:
This seems like a lot of code for very little abstraction. Why not just handle the focus event directly in the `RFB` class?